digital_asset_types/src/rpc/asset.rs

```diff
@@ -391,6 +391,17 @@ pub struct TokenInscriptionInfo {
     pub validation_hash: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct TokenInfo {
+    pub supply: u64,
+    pub decimals: u8,
+    pub token_program: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub mint_authority: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub freeze_authority: Option<String>,
+}
```

Comment:
I think Helius is also adding in token price in USDC here.  I know that's like some kind of extra Oracle lookup at query time, but just wanted to call it out as an FYI.

```
"token_info" : {
         "decimals" : 9,
         "mint_authority" : "6iQKfEyhr3bZMotVkW6beNZz5CPAkiwvgV2CTje9pVSS",
         "price_info" : {
            "currency" : "USDC",
            "price_per_token" : 247.02798
         },
         "supply" : 12572360539785052,
         "symbol" : "JitoSOL",
         "token_program" : "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"
}
```

Reply:
We noted this but if its requires third party API call I wanted to leave out. I'd personally like everything be from the chain.